src/TwigExtension.php

```diff
@@ -15,7 +15,7 @@ public function getGlobals()
     {
         return [
             'upper' => [
-                'cache' => \Craft::createObject(CacheResponse::class)
```

Comment:
The idea behind createObject() was to leverage DI, I guess. >> automatically resolving the `Response` when constructing `CacheResponse`. Not sure if it actually works.